bootstrap.php

```diff
@@ -20,4 +20,6 @@
     throw new \RuntimeException('Could not read config.php, please copy config.php-DEVELOPMENT to config.php & customize to your needs!');
 }
 
+class_alias('PHPUnit\Framework\TestCase', 'PHPUnit_Framework_TestCase');
```

Comment:
Maybe add comment that this is for BC for code written using old form?

Reply:
Is this even needed? What dependencies are using non namespaced variant of this class?

Reply:
> What dependencies are using non namespaced variant of this class?

Kernel tests are executed from meta-repo also, so we need to check if that works anyway.